src/testrender/optixraytracer.cpp

```diff
@@ -204,37 +208,45 @@ OptixRaytracer::init_optix_context (int xres OSL_MAYBE_UNUSED,
 
     if (scene.num_prims()) {
-        m_optix_ctx["radiance_ray_type"]->setUint  (0u);
-        m_optix_ctx["shadow_ray_type"  ]->setUint  (1u);
-        m_optix_ctx["bg_color"         ]->setFloat (0.0f, 0.0f, 0.0f);
-        m_optix_ctx["bad_color"        ]->setFloat (1.0f, 0.0f, 1.0f);
+        m_optix_ctx["radiance_ray_type"]->setUint(0u);
+        m_optix_ctx["shadow_ray_type"]->setUint(1u);
+        m_optix_ctx["bg_color"]->setFloat(0.0f, 0.0f, 0.0f);
+        m_optix_ctx["bad_color"]->setFloat(1.0f, 0.0f, 1.0f);
 
         // Create the OptiX programs and set them on the optix::Context
         if (renderer_ptx.size()) {
-            m_optix_ctx->setMissProgram (0, m_optix_ctx->createProgramFromPTXString (renderer_ptx, "miss"));
-            m_optix_ctx->setExceptionProgram (0, m_optix_ctx->createProgramFromPTXString (renderer_ptx, "exception"));
+            m_optix_ctx->setMissProgram(
+                0,
+                m_optix_ctx->createProgramFromPTXString(renderer_ptx, "miss"));
+            m_optix_ctx->setExceptionProgram(
+                0, m_optix_ctx->createProgramFromPTXString(renderer_ptx,
+                                                           "exception"));
         }
 
         // Load the PTX for the wrapper program. It will be used to
         // create OptiX Materials from the OSL ShaderGroups
-        m_materials_ptx = load_ptx_file ("wrapper.ptx");
+        m_materials_ptx = load_ptx_file("wrapper.ptx");
         if (m_materials_ptx.empty())
             return false;
 
         // Load the PTX for the primitives
-        std::string sphere_ptx = load_ptx_file ("sphere.ptx");
-        std::string quad_ptx = load_ptx_file ("quad.ptx");
+        std::string sphere_ptx = load_ptx_file("sphere.ptx");
+        std::string quad_ptx   = load_ptx_file("quad.ptx");
         if (sphere_ptx.empty() || quad_ptx.empty())
             return false;
 
         // Create the sphere and quad intersection programs.
-        sphere_bounds    = m_optix_ctx->createProgramFromPTXString (sphere_ptx, "bounds");
-        quad_bounds      = m_optix_ctx->createProgramFromPTXString (quad_ptx,   "bounds");
-        sphere_intersect = m_optix_ctx->createProgramFromPTXString (sphere_ptx, "intersect");
-        quad_intersect   = m_optix_ctx->createProgramFromPTXString (quad_ptx,   "intersect");
+        sphere_bounds    = m_optix_ctx->createProgramFromPTXString(sphere_ptx,
+                                                                   "bounds");
+        quad_bounds      = m_optix_ctx->createProgramFromPTXString(quad_ptx,
+                                                                   "bounds");
+        sphere_intersect = m_optix_ctx->createProgramFromPTXString(sphere_ptx,
+                                                                   "intersect");
+        quad_intersect   = m_optix_ctx->createProgramFromPTXString(quad_ptx,
+                                                                   "intersect");
     }
```

Comment:
This was more readable prior to clang-format.

Reply:
meh, gonna leave